### DISCUSSION TITLE
does it exist a way to save [xournal file + pdf] in a single file?

### DISCUSSION THREAD

**Discussion by Samanbaia** [xournalpp/xournalpp#5370] [General]
does it exist a way to save **[xournal file + pdf**] in a single file?

Thanks :-)

**Comment by rolandlo** [comment]
Currently not, although this is worked on with the new file format.

**Reply by nbruin** [reply]
This would be very desirable. The current dependence on the pdf file remaining at an absolute pathname or relying on a relative pathname and the user making sure the two files are copied together is very fragile. For instance, in a setup where the PDF and the xopp live on a Dropbox-managed directory (or something similar), the absolute file-names already break (very fundamentally if the folder is shared between Windows and Linux, where the absolute filenames fundamentally look different). Do we have a timeline on the new file format? Can I help in expediting the delivery of the feature?
